### PR TITLE
Add playbook file path checking

### PR DESCRIPTION
**Pull request by ushergod**
Signed-off-by: Sihan Wang <wsihan@vmware.com>

### CODE REVIEW

File: column/api/manager/run_manager.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import threading
 
 import column
+from column import exceptions
```
Comment (ericwb) [inline column/api/manager/run_manager.py]: alpha sort. e comes after a

File: column/exceptions.py [hunk 2]
```diff
@@ -31,3 +31,8 @@ def __init__(self, msg=None, **kwargs):
 class InvalidParameter(BaseException):
     """Invalid parameter given"""
     message = "Invalid type of %(name)s on parameter %(param)s"
+
+
+class FileNotFound(BaseException):
+    """Given file can not be found"""
```
Comment (ericwb) [inline column/exceptions.py]: cannot is one word, no space

File: column/exceptions.py [hunk 3]
```diff
@@ -31,3 +31,8 @@ def __init__(self, msg=None, **kwargs):
...
+    message = "File %(name)s can not be found"
```
Comment (ericwb) [inline column/exceptions.py]: cannot is one word, no space

File: tests/api/controllers/test_run.py [hunk 4]
```diff
@@ -109,3 +109,17 @@ def test_trigger_run(self):
...
+    def test_invalid_filepath(self):
+        pb = 'tests/fixtures/playbooks/not-exist.yml'
+        response = self.app.post(
+            '/runs',
+            data=json.dumps(dict(playbook_path=pb,
+                                 inventory_file='localhost,',
+                                 options={'connection': 'local'})),
+            content_type='application/json')
+        res_dict = json.loads(response.data)
+        time.sleep(2)
```
Comment (ericwb) [inline tests/api/controllers/test_run.py]: why the sleep?  usually this will cause problems where the timing of system runs differently.

Comment (ushergod) [inline tests/api/controllers/test_run.py]: My goal is to make sure the run fails before GET method comes in.

**Pull request merged**